app/services/offer_validations.rb

```diff
@@ -99,7 +99,9 @@ def reference_condition
   end
 
   def any_accepted_offers?
-    (application_choice.self_and_siblings - [application_choice]).map(&:status).map(&:to_sym).intersect?(ApplicationStateChange::ACCEPTED_STATES)
+    (application_choice.self_and_siblings - [application_choice])
+      .map(&:status).map(&:to_sym)
+      .intersect?(ApplicationStateChange::ACCEPTED_STATES - [:conditions_not_met])
```

Comment:
This method doesn't make sense anymore. `conditions_not_met` is and accepted offer state. The method name says **any**_accepted_offers?

Maybe we should create a method somewhere `can_receive_other_offers?` or something along those lines?

Reply:
Updated here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/9141/commits/f29c27fcba8b0e2f4d1f663a802ee06e0298feb4